lib/pipeline.ml

```diff
@@ -89,22 +110,8 @@ let docker_specs ~analysis =
             ~analysis (`Lint `Fmt)
           :: Spec.opam_monorepo builds
       | `Opam_build selections ->
-          let selections =
-            (* TODO: This filter will be removed when a new version of debian in which capnproto has this fix "https://github.com/capnproto/capnproto/pull/1830".
-               * Related to this issue "https://github.com/mirage/capnp-rpc/issues/273" *)
-            List.filter
-              (fun s ->
-                not
-                  (Variant.arch s.Selection.variant == `I386
-                  && Variant.arch s.Selection.variant == `Aarch32
```

Comment:
Unless I am missing something, this filter could never match anything as is, since it is impossible for the same value to `==` both of these variants.